explore-analyze/elastic-inference/inference-api.md

```diff
@@ -93,11 +93,15 @@ By default, documents are split into sentences and grouped in sections up to 250
 
 ### Chunking strategies
 
-Two strategies are available for chunking: `sentence` and `word`.
+Four strategies are available for chunking: `sentence`, `word`, `recursive` and `none`.
 
-The `sentence` strategy splits the input text at sentence boundaries. Each chunk contains one or more complete sentences ensuring that the integrity of sentence-level context is preserved, except when a sentence causes a chunk to exceed a word count of `max_chunk_size`, in which case it will be split across chunks. The `sentence_overlap` option defines the number of sentences from the previous chunk to include in the current chunk which is either `0` or `1`.
+- The `sentence` strategy splits the input text at sentence boundaries. Each chunk contains one or more complete sentences ensuring that the integrity of sentence-level context is preserved, except when a sentence causes a chunk to exceed a word count of `max_chunk_size`, in which case it will be split across chunks. The `sentence_overlap` option defines the number of sentences from the previous chunk to include in the current chunk which is either `0` or `1`.
 
-The `word` strategy splits the input text on individual words up to the `max_chunk_size` limit. The `overlap` option is the number of words from the previous chunk to include in the current chunk.
+- The `word` strategy splits the input text on individual words up to the `max_chunk_size` limit. The `overlap` option is the number of words from the previous chunk to include in the current chunk.
+
+- {applies_to}`stack: ga 9.1` The `recursive` strategy splits the input text based on a configurable list of separator patterns (for example, newlines or Markdown headers). The chunker applies these separators in order, recursively splitting any chunk that exceeds the `max_chunk_size` word limit. If no separator produces a small enough chunk, the strategy falls back to sentence-level splitting.
```

Comment:
@shainaraskas @florent-leborgne very aesthetic question: I think inline applies_to's should be placed at the end of a bullet, because the actual information is primary and the metadata is secondary, but maybe this is something that we intend to rely on the docs-build to standardize in the future

Reply:
I think that by putting it at the start then it's more obvious that it's for the entire bullet. Point taken though, inline badge UX is part of upcoming explorations, which is why there is no official guidance at the moment for this.